server/src/main/java/org/elasticsearch/cluster/service/TransportVersionsFixupListener.java

```diff
@@ -98,26 +103,26 @@ public TransportVersionsFixupListener(
     }
 
     class NodeTransportVersionTask implements ClusterStateTaskListener {
```

Comment:
Maybe rename the class?

Reply:
`NodeCompatibilityVersionsTask`?
Should I rename the outer class too, to `CompatibilityVersionsFixupListener`?

Reply:
Yes, that would make sense